### PR TITLE
After success method

### PR DESCRIPTION
**Pull request by jakub-dzon**
This PR provides method to be executed after a successful VM import.

```release-note
NONE
```
Signed-off-by: Jakub Dzon <jdzon@redhat.com>

### CODE REVIEW

File: pkg/controller/virtualmachineimport/virtualmachineimport_controller.go [hunk 1]
```diff
@@ -381,6 +381,42 @@ func (r *ReconcileVirtualMachineImport) storeSourceVMStatus(instance *v2vv1alpha
...
 }
 
+//TODO: use in proper place
+func (r *ReconcileVirtualMachineImport) afterSuccess(vmName types.NamespacedName, vmiName types.NamespacedName, p provider.Provider) error {
+	var errs []error
+	err := r.afterImport(p, vmiName.Namespace)
+	if err != nil {
+		errs = append(errs, err)
+	}
+	err = r.removeOwnerReference(vmName)
```
Comment (pkliczewski) [inline pkg/controller/virtualmachineimport/virtualmachineimport_controller.go]: We need to remove `ownerReference` for data volumes, configMaps and secrets.

Comment (jakub-dzon) [reply pkg/controller/virtualmachineimport/virtualmachineimport_controller.go]: The temporary map and secret are deleted (`afterImport` call) and for dv we don't set the owner reference (at least not yet). I will add the reference removal, though.

Comment (machacekondra) [reply pkg/controller/virtualmachineimport/virtualmachineimport_controller.go]: Owner references are set in this PR for dv: https://github.com/kubevirt/vm-import-operator/pull/51 so would be nice if you add also cleanup of owner references for DV

Comment (jakub-dzon) [reply pkg/controller/virtualmachineimport/virtualmachineimport_controller.go]: Will do!

Comment (jakub-dzon) [reply pkg/controller/virtualmachineimport/virtualmachineimport_controller.go]: Done.

**Review by pkliczewski — APPROVED** [review]

**Review by machacekondra — APPROVED** [review]

**Pull request merged**